editor.planx.uk/src/@planx/components/Review/Public.test.tsx

```diff
@@ -103,3 +110,210 @@ it("should not have any accessibility violations", async () => {
   const results = await axe(container);
   expect(results).toHaveNoViolations();
 });
+
+const mockLink = "my-file.png";
+const uploadedPlanUrl = "http://someurl.com/whjhnh65/plan.png";
+
+beforeEach(() => {
+  global.URL = {
+    createObjectURL: jest.fn(() => mockLink),
+  } as any;
+});
+
+it("should render file upload link", async () => {
```

Comment:
nit: "link" feels a bit confusing here because it's an important change in this PR that these are actually _NOT_ hyperlinks anymore in the sense of HTML tags. can "link" be updated to "filename" or something similar please?